src/common/env-and-consts.ts

```diff
@@ -19,6 +19,17 @@ export const SUPPORTED_LOCALES: ReadonlySet<string> = new Set([
   SupportedLocale.JA_JP,
 ]);
 
+export const RETAINED_LOCALSTORAGE_KEYS = [
```

Comment:
This should go to the util file because it's no longer shared between main and renderer